src/CPlusPlusMangle.cpp

```diff
@@ -29,10 +29,10 @@ struct MangledNamePart {
     std::string with_substitutions;
 
     MangledNamePart() = default;
-    MangledNamePart(const std::string &mangled)
+    explicit MangledNamePart(const std::string &mangled)
         : full_name(mangled), with_substitutions(mangled) {
     }
-    MangledNamePart(const char *mangled)
+    explicit MangledNamePart(const char *mangled)
```

Comment:
I think this one would be better as implicit

Reply:
PTAL. I reverted the changes and added the magic comment that would disable the clang-tidy check if we decided to actually enforce it. (Yes, it's pretty ugly.)

Honestly, we depend a *lot* on implicit conversions in a lot of places, so if decided to turn on this check we'd have a lot of places marked as exempt. It would let us avoid undesired implicit conversions for new code, though.